tests/test_base_scalers.py

```diff
@@ -121,3 +121,14 @@ def test_runs_query_if_failure_is_old(self, mock_db_connection):
         self.db_query_scaler.run_query()
 
         assert mock_db_connection.called is True
+
+
+@freeze_time("2018-01-01 12:00")
+class TestDbQueryScalerWithFixedCredentials:
+    def setup_method(self, method):
+        with patch.dict(os.environ, {"SQLALCHEMY_DATABASE_URI": "test-db-uri"}):
```

Comment:
I'd suggest this becomes `postgresql://test-db-uri` so that we test the logic about replacing `postgresql` with `postgres`. 

This suggestion is strongly encouraged, rather than optional. This is because the lack of explicit test made me wonder if you had made a mistake with your argument order in the str.replace function, because you hadn't explicitly called out in a python test what behaviour you were expecting to see.

Reply:
Updated